### PR TITLE
PSR-804 Added bonds disposal list page

### PR DESCRIPTION
**Pull request by westwater**

### CODE REVIEW

File: app/views/ListRadiosView.scala.html [hunk 1]
```diff
@@ -46,7 +46,7 @@ <h1 class="govuk-heading-l">@renderMessage(viewmodel.heading)</h1>
                   field = form("value"),
                   items = rows,
                   legend = viewmodel.page.legend
-                    .map(legend => Some(LegendViewModel(renderMessage(legend)).withSize(LegendSize.Small)))
+                    .map(legend => Some(LegendViewModel(renderMessage(legend)).withSize(LegendSize.Medium)))
```
Comment (kberec) [inline app/views/ListRadiosView.scala.html:49]: This is going to affect all pages using this view, including a choice page for removal (`WhichEmployerContributionRemoveController`), just wanted to double check if this is ok.

File: test/controllers/nonsipp/bondsdisposal/BondsDisposalListControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    ),
+    BondsDisposalData(
+      shareIndexThree,
+      nameOfBonds = "bond threee",
+      heldBondsType = SchemeHoldBond.Contribution,
+      bondsValue = money
+    )
+  )
+
+  "SharesDisposalListControllerSpec" - {
```
Comment (kberec) [inline test/controllers/nonsipp/bondsdisposal/BondsDisposalListControllerSpec.scala:79]: Share -> Bond

File: test/controllers/nonsipp/bondsdisposal/BondsDisposalListControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  IncomeFromBondsPage,
+  NameOfBondsPage,
+  WhyDoesSchemeHoldBondsPage
+}
+import views.html.ListRadiosView
+
+class BondsDisposalListControllerSpec extends ControllerBaseSpec {
+
+  private val page = 1
+  private val shareIndexOne = refineMV[Max5000.Refined](1)
```
Comment (kberec) [inline test/controllers/nonsipp/bondsdisposal/BondsDisposalListControllerSpec.scala:36]: Share -> Bond

File: app/controllers/nonsipp/bondsdisposal/BondsDisposalListController.scala [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+  val controllerComponents: MessagesControllerComponents,
+  view: ListRadiosView,
+  formProvider: RadioListFormProvider
+) extends PSRController {
+
+  val form: Form[Max5000] = BondsDisposalListController.form(formProvider)
+
+  def onPageLoad(srn: Srn, page: Int, mode: Mode): Action[AnyContent] = identifyAndRequireData(srn) {
+    implicit request =>
+      // todo swap when BondsCompleted has been added
```
Comment (kberec) [inline app/controllers/nonsipp/bondsdisposal/BondsDisposalListController.scala:63]: FYI @B-Savage there may be some cross-over with your ticket and bonds completed  page

File: app/navigation/nonsipp/BondsDisposalNavigator.scala [hunk 5]
```diff
@@ -34,6 +34,9 @@ object BondsDisposalNavigator extends JourneyNavigator {
       }
 
     case WhatYouWillNeedBondsDisposalPage(srn) =>
+      controllers.nonsipp.bondsdisposal.routes.BondsDisposalListController.onPageLoad(srn, 1, NormalMode)
```
Comment (kberec) [inline app/navigation/nonsipp/BondsDisposalNavigator.scala:37]: Do we need a test?

File: app/views/ListRadiosView.scala.html (hunk 1 above)
Comment (westwater) [reply app/views/ListRadiosView.scala.html:49]: I'll pull this out into a param

Comment (kberec) [reply app/views/ListRadiosView.scala.html:49]: Only if it's needed @westwater . If all usages need to be Medium, please keep it

**Review by kberec — APPROVED** [review]

**Pull request merged**